.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -525,6 +525,21 @@ stages:
           testResultsFiles: tracer/build_data/results/**/*.trx
         condition: succeededOrFailed()
 
+      - task: NodeTool@0 
+        inputs:
+          versionSpec: '12.x'
+
+      - task: PowerShell@2
+        displayName: Upload JUnit results
+        inputs:
+          targetType: 'filePath'
+          filePath: .azure-pipelines/upload-junit.ps1
+        env:
+          DD_API_KEY: $(ddApiKey)
+          DD_ENV: ci
+          ARCHITECTURE: x64
+        condition: succeededOrFailed()
+
```

Comment:
There's a lot of duplication here, we could we extract these to a template. That way we can tear out the powershell/node again once the standalone runner supports arm64?